locator.go

```diff
@@ -30,7 +30,7 @@ func newLocator(frame *frameImpl, selector string, options ...LocatorLocatorOpti
 			if frame != has.frame {
 				return nil, errors.New("inner 'has' locator must belong to the same frame")
 			}
-			selector += " >> has=" + has.selector
+			selector += fmt.Sprintf(` >> has="%s"`, has.selector)
```

Comment:
Lets do it maybe the same was as upstream? https://github.com/microsoft/playwright/blob/634ba85c83ae19de1e1610e5b393e35eb03012e4/packages/playwright-core/src/client/locator.ts#L54

Reply:
sorry for the late reply, I have some urgent matters. Thanks for the comment, I will present a new version in the coming days

Reply:
@mxschmitt Yes, I think it was a good idea. I replaced fmt.Sprintf with  json.Marshal. Could you take a look one more time?